### ISSUE TITLE
[FEATURE REQUEST] Widget for twitter/x timeline, specific list, communities features

### ISSUE THREAD

**Issue by HummusB0x** [glanceapp/glance#93]
Could it be possible to add a Twitter / X widget please?

This could be implemented step by step:
- Personal `timeline`
- Personal `lists`
- Specific `communities`

A bit like the good old **tweetdeck** application. 

I am aware that this would require to handle a session token but could be still quite interesting for your awesome tool.

Maybe there would an alternative solution where we convert X feeds in RSS so that this process is not handled by your tool. Are you aware of such project?

**Comment by helloteemo** [comment]
@HummusB0x Yes, there are projects like this, [RSSHub](https://docs.rsshub.app/)
